docs/docs/visual-testing-with-storybook.md

```diff
@@ -11,22 +11,15 @@ As your project grows over time having this information available will be invalu
 
 ## Setting up your environment
 
-To set up Storybook you need to install dependencies and do some custom configuration. First, install the Storybook CLI.
+To set up Storybook you need to install dependencies and do some custom configuration. You can get started quickly by using the automated command line tool from your Gatsby root directory:
 
 ```shell
-npm install -g @storybook/cli
+npx -p @storybook/cli sb init
 ```
 
```

Comment:
@laurieontech and @chrish-d probably go with something like:
"As you may have noticed we're using `npx` instead of `npm` to setup Storybook. Amongst others, the two main reasons behind this are the following:

- Ensure you always have the latest Storybook version added to your project.
- Also automatically generate a file/folder structure with the default configuration for you.
"
Which is what is actually happening when the reader runs the command.

Feel free to provide feedback

Reply:
I'm wondering if something like this might be effective:

"> Note that the following instructions are using `npx`. [npx](https://www.npmjs.com/package/npx) is part of npm and in this case it allows you to automatically generate a file/folder structure complete with the default configuration."

Reply:
Yeah I think @laurieontech's suggestion works. If a user wants to learn more about `npx` they can follow the link, rather than cluttering up the docs with further explanation.

Reply:
fine by me, lean, consise and effective. Hoping this gets merged so that the related pr continues his work and possibly his pr is merged and he gets his swag aswell :)

Reply:
So my main outstanding question is whether we feel it's ok to merge this without the document overhaul? That was the initial concern.

Reply:
@laurieontech hence my original comment in the other pr. From what i see probably the safest bet is to merge this, and in the other pr update it with this and go from there like i mentioned in [here](https://github.com/gatsbyjs/gatsby/pull/21378#issuecomment-585423848)

Reply:
Precisely. Just wanted to make sure that was still what made sense to everyone.